Dockerfile

```diff
@@ -45,6 +45,10 @@ RUN curl -o /usr/bin/rancherd -sfL "https://github.com/rancher/rancherd/releases
 ARG VIRTCTL_VERSION=v0.55.2
 RUN curl -o /usr/bin/virtctl -sfL "https://github.com/kubevirt/kubevirt/releases/download/${VIRTCTL_VERSION}/virtctl-${VIRTCTL_VERSION}-linux-amd64" && chmod 0755 /usr/bin/virtctl
 
+# Download yip
+ARG YIP_VERSION=v1.0.0
```

Comment:
wondering why we skip install via luet?

Reply:
New versions are not published to luet repo anymore.

Reply:
Yap, for the short term, we plan to download directly.
For the long term, we may try to maintain our package list on OBS.